internal/plugin/server.go

```diff
@@ -422,6 +422,14 @@ func (plugin *NvidiaDevicePlugin) updateResponseForCDI(response *pluginapi.Conta
 		}
 		response.Annotations = annotations
 	}
+	if plugin.deviceListStrategies.Includes(spec.DeviceListStrategyCDICRI) {
```

Comment:
One note: It is probably not recommended to set *both* `cdi-annotations` and `cdi-cri` since the latter also sets annotations if enabled in the kubelet.